src/Test/L1/Mock/FakeReleaseServer.cs

```diff
@@ -0,0 +1,41 @@
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System;
+using Microsoft.VisualStudio.Services.WebApi;
+using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi.Contracts;
+using Microsoft.VisualStudio.Services.Agent.Worker.Release;
+using RMContracts = Microsoft.VisualStudio.Services.ReleaseManagement.WebApi;
+
+namespace Microsoft.VisualStudio.Services.Agent.Tests.L1.Worker
+{
+    public class FakeReleaseServer : AgentService, IReleaseServer
+    {
+        public string ReleaseName { get; internal set; }
+
+        public Task ConnectAsync(VssConnection jobConnection)
+        {
+            return Task.CompletedTask;
+        }
+
+        public IEnumerable<AgentArtifactDefinition> GetReleaseArtifactsFromService(
+            int releaseId,
+            Guid projectId,
+            CancellationToken cancellationToken = default(CancellationToken))
+            {
```

Comment:
Nit: formatting is off here and need extra line between methods.

Reply:
Fixed